md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ProcessTemplateAPI.java

```diff
@@ -84,13 +84,13 @@ RestWrapper get(
                 processTemplate.setBatchPattern(jpaProcessTemplate.getBatchCutPattern());
                 processTemplate.setNextProcessTemplateId(jpaProcessTemplate.getNextProcessTemplateId());
             }
-            //  processTemplate = s.selectOne("call_procedures.GetProcessTemplate", processTemplate);
             processTemplate.setTableAddTS(DateConverter.dateToString(processTemplate.getAddTS()));
 
             restWrapper = new RestWrapper(processTemplate, RestWrapper.OK);
             LOGGER.info("Record with ID:" + processTemplateId + " selected from ProcessTemplate by User:" + principal.getName());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Record with ID:" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)